models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql

```diff
@@ -0,0 +1,64 @@
+{{config(alias='aave_v2_liquidations')}}
+
+
+WITH mev_addresses AS (
+    SELECT 
+        address
+    FROM {{ ref('labels_mev_ethereum') }}
+)
+
+, aave_v2_annual_liquidations AS (
+    SELECT 
+        * 
+    FROM {{ source('aave_v2_ethereum', 'LendingPool_evt_LiquidationCall') }}
+    WHERE evt_block_time > now() - INTERVAL '365' day
+)
+
+, all_liquidations AS (
+SELECT
+    a.evt_block_time
+    , a.evt_tx_hash
+    , a.collateralAsset
+    , cast(a.liquidatedCollateralAmount as double) / pow(10, p.decimals) AS liquidatedCollateralAmount
+    , p.symbol
+    , m.address
+    , m.address AS liquidator
+FROM aave_v2_annual_liquidations a
+LEFT JOIN mev_addresses m ON m.address = a.liquidator
+LEFT JOIN prices.tokens p ON p.contract_address = a.collateralAsset
```

Comment:
please use jinja syntax here for a source

Reply:
Hi @jeff-dude 

The `FROM aave_v2_annual_liquidations a` reference on line 26 refers to a previous CTE within the same .sql file on line 10 (it coincidentally has the same name as the file, if that's confusing, I can change the .sql file name itself)

In this scenario, is jinja syntax required? 

if yes, i'm thinking it would it be something like: 

`FROM {{ aave_v2_annual_liquidations }} a`   -- would this be the way to go?

Thank you.

Reply:
i'm referring to `prices.tokens` as a source, not the `aave_v2_annual_liquidations` CTE

Reply:
oops, meant ref, not source. `prices.tokens` is a spell, while `prices.usd` is a source. i misread that.